server/controllers/postCustomer.js

```diff
@@ -2,9 +2,9 @@ const postCustomer = require('../database/queries/postCustomer');
 
 module.exports = (req, res) => {
   const data = req.body;
-  const { fullName, phone, email, zipCode, reserveDate } = data;
+  const { fullName, phone, email, zipCode, reservationDate, timeId } = data;
   // check if the all text fields is filled.
-  if (!fullName || !phone || !email || !zipCode || !reserveDate) {
+  if (!fullName || !phone || !email || !zipCode || !reservationDate || !timeId) {
     // if one of text fields is empty, show this message
     return res.status(400).json({
       message: 'There is one item or more missing. Please filled it!',
```

Comment:
This error message can be more discriptive, for instance you may say:
`Please make sure that you filled each field properly` 
still believe that this is not good so maybe we can change it later.

Reply:
descriptive :stuck_out_tongue_winking_eye: